e2e/tables.spec.ts

```diff
@@ -129,21 +129,24 @@ test.describe('Tables', () => {
       reorder_quantity: '25',
     }))
 
-    await test.step('User goes to tables page', async () => {
+    await testAuth.step('User goes to tables page', async () => {
       await goToDashboardPage(userPage, getTablesPath(userProject.id))
       await expectTablesPageLoaded(userPage, userProject.id)
       await expect(userPage.getByText(noTablesText)).toBeVisible()
     })
 
     // TODO: remove this step after table descriptions are visible outide of experimental mode
-    await test.step('Toggle into Experimental Mode to use react-based table column schema editor', async () => {
-      await toggleIntoExperimentalMode(userPage)
-      await expectTablesPageLoaded(userPage, userProject.id)
-      await expect(userPage.getByText(noTablesText)).toBeVisible()
-    })
+    await testAuth.step(
```

Comment:
The rest of the changes in this file are 1) changing `test` -> `testAuth` and 2) associated whitespace/formatting changes.

Reply:
*Other than the specific change to the empty table expectation, as noted below